### PR TITLE
Update Intel Aero information

### PR DESCRIPTION
**Pull request by zehortigoza**

### CODE REVIEW

File: en/flight_controller/intel_aero.md [hunk 1]
```diff
@@ -46,7 +48,11 @@ After setting up the PX4 development environment, follow these steps update the
    If it doesn't work you can try giving the IP that will be used by the upload script:
 
    ```
-   export AERO_HOSTNAME=192.168.1.1`
+   # WiFi IP
+   export AERO_HOSTNAME=192.168.8.1`
```
Comment (hamishwillee) [inline en/flight_controller/intel_aero.md]: Extra backtick at the end of this line.

File: en/flight_controller/intel_aero.md [hunk 2]
```diff
@@ -46,7 +48,11 @@ After setting up the PX4 development environment, follow these steps update the
...
+
+   # Ethernet-over-USB IP
+   export AERO_HOSTNAME=192.168.7.2`
```
Comment (hamishwillee) [inline en/flight_controller/intel_aero.md]: Extra backtick at the end of this line.

**Comment by hamishwillee** [comment]
Thanks @zehortigoza  ! Looks good except for the minor inline comments.

**Comment by zehortigoza** [comment]
@hamishwillee Oh sorry, it is fixed.
Thanks

File: en/flight_controller/intel_aero.md [hunk 3]
```diff
@@ -23,8 +23,10 @@ get_aero_version.py
 The instructions here are tested with the following version:
 
 ```
-BIOS_VERSION = Aero-01.00.12_Prod
-OS_VERSION = v01.00.04
+BIOS_VERSION = Aero-01.00.13
+OS_VERSION = Poky Aero (Intel Aero Linux Distro) 1.5.1-dev (pyro)"
```
Comment (hamishwillee) [inline en/flight_controller/intel_aero.md]: Is the " at the end intentional?

**Comment by hamishwillee** [comment]
Great. Just added one more comment. I'm off on holiday for a week - @bkueng should be able to merge this.

File: en/flight_controller/intel_aero.md (hunk 3 above)
Comment (zehortigoza) [reply en/flight_controller/intel_aero.md]: Oh is not intentional but is the output that get_aero_version.py prints, so better keep in the documentation until fixed in the Linux image.

**Comment by hamishwillee** [comment]
Thanks, all done!

**Pull request merged**